nowplaying/input/observer.py

```diff
@@ -120,8 +126,8 @@ def get_track_info(self):
 
         # default track info
         track_info = {
-            "artist": track.track.DEFAULT_ARTIST,
-            "title": track.track.DEFAULT_TITLE,
+            "artist": DEFAULT_ARTIST,
```

Comment:
These are still from `track.track` (but via import at the top) since I decided that refactoring `track.track` to not expose these is outside of the scope of this pr. I did the `_SHOW` vars in this PR since there were no blockers with them and to make exactly how this line works less easy to implement (I consider calling constants in an external module that isn't a explicit "constants" or config module rather hacky).